.github/resources/opencompass-hf-results.json

```diff
@@ -1,14 +1,14 @@
 {
     "meta-llama/Llama-2-7b-chat": {
-        "ceval": "28.3",
-        "mmlu": "35.49",
+        "ceval": "30.44",
```

Comment:
This is the evaluation ground truth by transformers inference, isn' t it?

Reply:
Some datasets like cmmlu, Gaokao and so on are added in `eval_opencompass_config.py`
Should the corresponding evaluation ground truth of the added datasets be updated in this json file?

Reply:
> This is the evaluation ground truth by transformers inference, isn' t it?

yes

Reply:
> Some datasets like cmmlu, Gaokao and so on are added in `eval_opencompass_config.py` Should the corresponding evaluation ground truth of the added datasets be updated in this json file?

need time to run them. I will do it.